app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,26 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/parking_walpaper_main"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_main_activity"
+        style="@style/TitleMainActivityStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/title_main"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/parkingSizeBtn"
```

Comment:
try to rename this id align the code style

Reply:
Done <3